manifest.webapp

```diff
@@ -270,6 +270,11 @@
       "description": "Used to update the document storing information entered before a transfer",
       "type": "io.cozy.identities",
       "verbs": ["GET", "PUT", "POST"]
+    },
+    "tags": {
+      "description": "Used to update the document storing information entered before a transfer",
+      "type": "io.cozy.tags",
+      "verbs": ["GET", "POST", "PUT", "DELETE"]
```

Comment:
Can you document it on https://github.com/cozy/cozy-doctypes?

Reply:
j'ai également changé la description : `Used to link tags to bank operations`

Reply:
je merge https://github.com/cozy/cozy-doctypes/pull/203 pour l'occasion ;)

Reply:
=> https://github.com/cozy/cozy-store/pull/801